apm-dist/release-docs/LICENSE

```diff
@@ -417,6 +417,7 @@ popper.js	1.14.7:	https://github.com/FezVrasta/popper.js	MIT
 vue-datepicker-local	1.0.19:	https://github.com/weifeiyue/vue-datepicker-local	MIT
 vue-js-modal	1.3.31:	https://github.com/euvl/vue-js-modal	MIT
 lodash	4.17.15:	https://github.com/lodash/lodash	MIT
+@types:	https://github.com/DefinitelyTyped/DefinitelyTyped	MIT
```

Comment:
Because @types does not have a version number, there is no version number filled in here.